index.js

```diff
@@ -165,7 +165,9 @@ class protractorImageComparison {
      */
     _saveFullScreenshot(tag, screens) {
         // Calculate total canvas size
-        const imageHeight = screens.reduce((previous, current) => (previous instanceof Buffer ? previous.readUInt32BE(20) : previous) + current.readUInt32BE(20));
+        const imageHeight = screens.reduce(
+            (previous, current) => (previous instanceof Buffer ? previous.readUInt32BE(20) : previous) + current.readUInt32BE(20),
+            screens[0].readUInt32BE(20));
```

Comment:
This will not work and provide incorrect image sizes when `screens.length > 1` because it will start the calculation with the initial size of the image.

This can be fixed with this, or maybe you have a better way ;-)

```js
let imageHeight;

if (screens.length > 1){
    imageHeight = screens.reduce(
        (previous, current) => (previous instanceof Buffer ? previous.readUInt32BE(20) : previous) + current.readUInt32BE(20)
    );
} else {
    imageHeight = screens[0].readUInt32BE(20)
}
```

Reply:
I tested (briefly, I admit :-)) with screens.length 2 and it worked. But I didn't go very deep into your code.

Your fix of course is fine as well. Go ahead as you like

Reply:
If you like you may add it and it will be your contribution, otherwise I'll fix it